sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/nested-not-in.sql

```diff
@@ -0,0 +1,198 @@
+-- Tests NOT-IN subqueries nested inside OR expression(s).
```

Comment:
https://github.com/apache/spark/pull/22141/files#r211835129
To make sure all the results are correct, I also checked that postgresql had the same output by the queries below.

Reply:
@maropu Oh.. thank you very much for trying out.